jac-uk/admin
# #1595 Character report pagination

**Author checklist**


---
## What's included?
Add pagination items to the bottom of the table. See [JAC Kit #57 Added pagination to bottom of table](https://github.com/jac-uk/jac-kit/pull/57). 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Preview URL: https://jac-admin-develop--pr1604-feature-1595-charact-smzkcf2z.web.app

- Make sure you use `@jac-uk/jac-kit` version `0.1.0`
- Open an exercise with character issues in it
- Check pagination functionality 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context
Demo:

![jac-admin-character-issues-pagination](https://user-images.githubusercontent.com/79906532/161984304-ae541599-e579-4f58-a439-9ed904308d66.gif)


## Related permissions
Have permissions been considered for this functionality?
- No permission changes required

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
